scripts/generate-babel-types-docs.js

```diff
@@ -4,7 +4,7 @@ var util = require('util');
 var path = require('path');
 var fs = require('fs');
 
-var types = require('../packages/babel-types');
+var types = require('../packages/babel-types/src');
```

Comment:
In one case you need to build the module first and in the other case you need to launch the script with ES6 support.

What do you prefer ?

We could use this technic : https://github.com/xtuc/CSS-as-React-component/blob/master/bin/index#L1.